cypress.json

```diff
@@ -1,4 +1,5 @@
 {
     "baseUrl": "http://localhost:8888",
-    "supportFile": false
+    "supportFile": false,
+    "failOnStatusCode": false
```

Comment:
Do we want this in? Unless there is a good reason to do so, we shouldn't be having Cypress bypass errors.